src/iris_api/api.py

```diff
@@ -2236,9 +2236,15 @@ def on_post(self, req, resp):
                 'status': post_dict['MessageStatus'],
                 'sid': post_dict['MessageSid']
```

Comment:
Do you think it's cleaner to avoid the exception handling with dictionary get method?

```python
sid = post_dict.get('MessageSid', post_dict.get('CallSid'))
status = post_dict.get('MessageStatus', post_dict.get('CallStatus'))
if not sid or not status:
    raise
```